inc/sai.h

```diff
@@ -147,10 +147,16 @@ typedef enum _sai_log_level_t
 
 } sai_log_level_t;
 
+/**
+* @brief Defines sai profile get value function pointer
+*/
```

Comment:
if you are doxygen adding comments, also params[] should be added with description, see other methods for examples and error logs when building